src/ui/components/Button/index.js

```diff
@@ -133,7 +136,8 @@ export default class Button extends React.Component<Props> {
     }
 
     return (
-      <button className={getClassName()} type="submit" {...props}>
+      // eslint-disable-next-line react/button-has-type
```

Comment:
it's a shame we can't create a `htmlType` type with only 3 possible values to satisfy this rule